src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -281,19 +281,18 @@ private void doRetrieve(SCMHeadObserver observer, TaskListener listener, GHRepos
             PullRequestSCMHead head = new PullRequestSCMHead(ghPullRequest);
             final String branchName = head.getName();
             listener.getLogger().format("%n    Checking pull request %s%n", HyperlinkNote.encodeTo(ghPullRequest.getHtmlUrl().toString(), "#" + branchName));
-            // FYI https://developer.github.com/v3/pulls/#response-1
-            Boolean mergeable = ghPullRequest.getMergeable();
-            if (!Boolean.TRUE.equals(mergeable)) {
-                listener.getLogger().format("    Not mergeable, skipping%n%n");
-                continue;
-            }
             if (repo.getOwner().equals(ghPullRequest.getHead().getUser())) {
                 listener.getLogger().format("    Submitted from origin repository, skipping%n%n");
                 continue;
             }
             if (criteria != null) {
                 SCMSourceCriteria.Probe probe = getProbe(branchName, "pull request", "refs/pull/" + head.getNumber() + "/head", repo, listener);
                 if (criteria.isHead(probe, listener)) {
+                    // FYI https://developer.github.com/v3/pulls/#response-1
+                    Boolean mergeable = ghPullRequest.getMergeable();
```

Comment:
According to the docs, "mergeable" is trinary - true, false or null, where null means "we haven't finished computing the mergeability yet". On larger repos, the webhook regularly fires before the mergeability has been computed, which leads to the plugin thinking every branch is unmergeable (even though they're not)